paymentsheet/src/main/java/com/stripe/android/paymentsheet/BaseAddPaymentMethodFragment.kt

```diff
@@ -118,6 +121,10 @@ internal abstract class BaseAddPaymentMethodFragment : Fragment() {
         }
 
         sheetViewModel.eventReporter.onShowNewPaymentOptionForm()
+
+        val isSystemDarkMode = context?.isSystemDarkTheme() ?: false
+        val surfaceColor = PaymentsThemeConfig.colors(isSystemDarkMode).surface
+        viewBinding.googlePayButton.setBackgroundColor(surfaceColor.shouldUseDarkDynamicColor())
```

Comment:
Are these changes allowed by the Google Pay icon requirements?  I think the .9. images are careful about stretching?